SPEC.md

```diff
@@ -153,7 +153,7 @@ public class Engine
 }
 ```
 
-the `init` keyword is a C# modifier which indicates that the property is only settable at construction time. In reality the Intermediate Language (IL) code just contains a regular setter and so long as properties in general have a getter and setter, the library will be able to use them. By default, all gettable properties will be serialized and all settable properties will be deserialized, but this can be configured as specified later in this document.
+the `init` keyword is a C# modifier which indicates that the property is only settable at construction time. In reality the Intermediate Language (IL) code just contains a regular setter and so long as properties in general have a getter and setter, the library will be able to use them. By default, all gettable properties will be serialized and all settable properties will be deserialized, but this can be configured as specified later in this document. The properties must have public, internal, or protected internal access modifier.
```

Comment:
Nit: "The properties must have a public, internal, ....."

Reply:
Or alternatively you could just pluralize "modifier" to "modifiers".

Reply:
Updated to "modifiers"